include/sisl/fds/buffer.hpp

```diff
@@ -270,6 +286,15 @@ struct io_blob : public blob {
     static io_blob from_string(const std::string& s) {
         return io_blob{r_cast< uint8_t* >(const_cast< char* >(s.data())), uint32_cast(s.size()), false};
     }
+
+    static io_blob_list_t sg_list_to_ioblob_list(sg_list const& sglist) {
```

Comment:
Again another styling comment: I think we can be consistent with 
`<Type> const& <variable>` or `const <Type>& <variable>`. I am ok with either, but same small class has 2 different style is little eyesore.